torch/_tensor_docs.py

```diff
@@ -4845,6 +4845,49 @@ def callable(a, b) -> number
 ``x.T`` is equivalent to ``x.permute(n-1, n-2, ..., 0)``.
 """)
 
+add_docstr_all('H',
+               r"""
+Returns a view of a matrix (2-D tensor) conjugated and transposed.
+
+``x.H`` is equivalent to ``x.transpose(0, 1).conj()`` for complex matrices and
+``x.transpose(0, 1)`` for real matrices.
+
+.. seealso::
+
+        :attr:`~.Tensor.mH`: An attribute that also works on batches of matrices.
+""")
+
+add_docstr_all('mT',
+               r"""
+Is this tensor with the last two dimensions transposed.
+
+``x.mT`` is equivalent to ``x.transpose(-2, -1)``.
+""")
+
+add_docstr_all('mH',
+               r"""
+Alias for :meth:`~Tensor.adjoint`.
+""")
+
+add_docstr_all('adjoint',
+               r"""
+adjoint(Tensor) -> Tensor
```

Comment:
index_fill() -> Tensor?

(this is part of why documenting the function is easier)